simple-methods/src/simple/TestOfSimpleMethods.java

```diff
@@ -0,0 +1,49 @@
+package simple;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class TestOfSimpleMethods {
+    @ParameterizedTest
+    @ValueSource(strings = {"012345", "qwert", "asdfgh", "íyxcvb"})
+    void isNotDuplicatedChars(String string){
+        Assertions.assertTrue(ThreeEasyMethods.isAllCharactersUnique(string));
+    }
+
+    @ParameterizedTest
+    @ValueSource(strings = {"00","qq","aa","cc", ",,"})
+    void isUniqueChars(String string){
+        Assertions.assertFalse(ThreeEasyMethods.isAllCharactersUnique(string));
+    }
+
+    @ParameterizedTest
+    @ValueSource(strings = {"papa mama baba mama papa mama", "PAPA MAMA MAMA PAPA MAMA BABA", "Papa paPA mAMA MaMa mamA BabA"})
+    void checkPiecesOfWords(String strings){
```

Comment:
By list of words I meant a list of strings, but we can say my description was ambiguous, so it is fine like this :)